test/tests/long.py

```diff
@@ -57,6 +57,10 @@ def test(a, b):
 print -(1L)
 print 1L**2L
 print 1L**2
+print 0 ** (1 << 100)
+print pow(1 << 30, 1 << 30, 127)
+print pow(1L << 30, 1L << 30, 127)
+print pow(1 << 100, 1 << 100, 1000)
```

Comment:
couple more tests:
```
print pow(-1, (1<<100))
print pow(-1, (1<<100) + 1)
print pow(0, (1<<100))
print pow(1, (1<<100))
print pow(5, 3, -7L)
print pow(-5,  3, 7L)
print pow(-5,  3, -7L)
```
(I don't think we had any tests that returned negative numbers)